skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/$templateId/-components/endreMottaker/HentOgVisAdresse.tsx

```diff
@@ -0,0 +1,121 @@
+import { css } from "@emotion/react";
+import { BodyShort, Heading, VStack } from "@navikt/ds-react";
+import { useQuery } from "@tanstack/react-query";
+
+import { getKontaktAdresse, getNavn, hentSamhandlerAdresse } from "~/api/skribenten-api-endpoints";
+import { ApiError } from "~/components/ApiError";
+import type { Adresse, KontaktAdresseResponse } from "~/types/apiTypes";
+import { getAdresseTypeName } from "~/types/nameMappings";
+
+import { Route } from "../../route";
+import { erAdresseKontaktAdresse } from "./EndreMottakerUtils";
+
+/**
+  En basic HentOgVis-komponent som henter og viser adresseinformasjon for en sak eller samhandler.
+ */
+const HentOgVisAdresse = (properties: { sakId: string; samhandlerId?: string; showMottakerTitle?: boolean }) => {
```

Comment:
Er HentOgVis et litt merkelig navn på et komponent? Jeg tenker da må kanskje alle komponenter som bruker react-query kalles for tilsvarende.

Reply:
I utgangspunktet er det ikke feil tanke det du sier. Det handler mest hvor nivået av hentingen gjøres. en vis/oppsummerings komponent skal i utgangspunktet ikke ha noe fetch logikk. Fetch logikken burde da gjøres i parenten. Men for å abstrahere fetch logikken som muligens parent gjør, kommer et mellomledd, som jeg ikke helt klarer å finne gode navn på. 

Typisk når du da leser kode fra parent sin side - vil du lese at her har vi en komponent som gjør en fetch + oppsummering